web/themes/custom/urbanobserver/components/00-base/00-defaults/_01-variables.scss

```diff
@@ -5,10 +5,16 @@
  * and breakpoint related vars into
  * components/00-base/base/_breakpoints.css
 */
-
+@import url('https://fonts.googleapis.com/css2?family=Big+Shoulders+Text:wght@100..900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Big+Shoulders+Text:wght@100..900&family=Work+Sans:ital,wght@0,100..900;1,100..900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Big+Shoulders+Inline+Text:wght@100..900&family=Big+Shoulders+Text:wght@100..900&family=Work+Sans:ital,wght@0,100..900;1,100..900&display=swap');
```

Comment:
Generally, assets like fonts are better to download and include from local files.
Not hard requirement but just FYI